apps/frontend/src/modules/users/users-api.ts

```diff
@@ -21,13 +22,19 @@ class UserApi extends BaseHTTPApi {
 		super({ baseUrl, http, path: APIPath.USERS, storage });
 	}
 
-	public async getAll(): Promise<UserGetAllResponseDto> {
+	public async getAll(
+		query: PaginationQueryParameters,
+	): Promise<UserGetAllResponseDto> {
 		const response = await this.load(
 			this.getFullEndpoint(UsersApiPath.ROOT, {}),
 			{
 				contentType: ContentType.JSON,
 				hasAuth: true,
 				method: "GET",
+				query: {
+					page: String(query.page),
+					pageSize: String(query.pageSize),
+				},
```

Comment:
I believe we don't need this after the change with typing above
```suggestion
				query
```